packages/matter.js/src/behavior/definitions/power-topology/PowerTopologyServer.ts

```diff
@@ -26,6 +27,8 @@ const PowerTopologyBase = PowerTopologyBehavior.with(
  */
 export class PowerTopologyServerLogic extends PowerTopologyBase {
     override initialize(): void {
+        this.agent.get(DescriptorServer).addDeviceTypes("ElectricalSensor");
```

Comment:
Formally check that a either Electrical Power Measurement or Electrical Energy Measurement cluster also exists on the endpoint because it is only allowed in this combination?

Reply:
Yes, good idea